### PR TITLE
Fix path to detect_ip.ps1 (DCOS_OSS-5662)

### PR DESCRIPTION
**Pull request by OleksandrBielov**
# [MEANINGFUL TITLE]
## Checklist
- [] ~80% unit test coverage?
- [] Updated [README.md](README.md)?
- [] Completed sections of this PR template?
- [] Edited all sections [IN BRACKETS]

## Overview of Change
[Why are you adding this? What problem does it solve?]

### Changelog [optional]
- [] [optional list]

## Affected Usage
[Description of how this change could affect usage of this library in other projects]

### CODE REVIEW

**Comment by OleksandrBielov** [comment]
@mesosphere-mergebot request-review @janisz

**Review by janisz — APPROVED** [review]

**Review by janisz — CHANGES_REQUESTED** [review]
Please fix description and tests

File: dcos/constants.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ const (
 func GetFileDetectIPLocation() string {
 	switch runtime.GOOS {
 	case "windows":
-		return "/opt/mesosphere/bin/detect_ip.ps1"
+		return "c:\\dcos\\opt\\bin\\detect_ip.ps1"
```
Comment (makkes) [inline dcos/constants.go:23]: Please add a unit test for this.

**Review by makkes — APPROVED** [review]
CI is failing, please fix.

**Review by makkes — CHANGES_REQUESTED** [review]
clicked "Approve" by accident

File: dcos/constants.go (hunk 1 above)
Comment (vadymzagorodko) [reply dcos/constants.go:23]: @makkes, the reason for this change is a request of @janisz on PR [https://github.com/dcos/dcos-diagnostics/pull/164] to update dcos-go with accordance with mentioned PR. Oleksandr Bielov isn't a GO developer, not sure that he should/can be responsible for the unit test

Comment (janisz) [reply dcos/constants.go:23]: Writing unit tests is essential part of writing the code.

Comment (DominikDary) [reply dcos/constants.go:23]: I agree with @janisz

**Comment by janisz** [comment]
@DominikDary @OleksandrBielov What's the status of this? Can we close it?

**Pull request closed without merging**